### DISCUSSION TITLE
Parsing JSON data for success / failure message

### DISCUSSION THREAD

**Discussion by darkorion6661** [healthchecks/healthchecks#838] [Q&A]
Hi all,

Looking at utilising Healthchecks.io for my private service notifications. I'm using Duplicacy for backups, and would like to trigger a notification to healthchecks.io if the backup fails. However, although I can POST the JSON data to healthchecks, it doesn't look like it can parse the data for success / failure information.

Is this something I'm overlooking? I can see we can configure parsing incoming emails, so I assumed we can do something similar for HTTP requests.

Cheers!

**Comment by cuu508** [comment]
No, Healthchecks currently does not inspect the HTTP request's body. 

I did play around with Duplicacy a bit ago, but didn't figure out a way to integrate it nicely with Healthchecks. I submitted [a feature request for `$?` special syntax in webhook URLs](https://forum.duplicacy.com/t/idea-evaluation-in-report-url/7246).
